usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -19,6 +20,96 @@
 ========
 """
 
+"""@pylatest api/gluster.cluster_create
+API-gluster: cluster_create
+***************************
+
+.. test_metadata:: author fbalak@redhat.com
+
+Description
+===========
+
+Positive create gluster cluster.
+"""
+
+
+def test_cluster_create_valid(valid_session_credentials, valid_nodes):
+    """@pylatest api/gluster.cluster_create
+        .. test_step:: 1
+
+        Check if there is at least one gluster node for cluster creation.
+
+        .. test_result:: 1
+
+        Test passes if there is at least one gluster node.
+        """
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    pytest.check(
+        len(valid_nodes) > 0,
+        "There have to be at least one gluster node."
+        "There are {}".format(len(valid_nodes)))
+    """@pylatest api/gluster.cluster_import
+        .. test_step:: 2
+
+            Send POST request to Tendrl API ``APIURL/GlusterCreateCluster
+
+        .. test_result:: 2
+
+            Server should return response in JSON format:
+
+                {
+                  "job_id": job_id
+                }
+
+            Return code should be **202**
+                with data ``{"message": "Accepted"}``.
+
+        """
+    nodes = []
+    provisioner_ip = None
+    for x in valid_nodes:
+        ips = ast.literal_eval(x["networks"]["eth0"]["ipv4"])
+        nodes.append({
+            "role": "glusterfs/node",
+            "ip": ips[0] if type(ips) == list else ips})
+        if "provisioner/gluster" in x["tags"]:
+            provisioner_ip = ips[0] if type(ips) == list else ips
+    LOGGER.debug("node_ips: %s" % nodes)
+    LOGGER.debug("provisioner: %s" % provisioner_ip)
+    job_id = api.create_cluster(
+            "MyCluster",
```

Comment:
The indentation here is not correct, it should be 4 characters not 8. pylint will detect this as an issue, flake8 doesn't mind though.
Anyway we should use the same indentation everywhere if possible. And even in this file there's 8 characters indentation here whilst later there's 4 characters used e.g. `integration_id = api.get_job_attribute(` etc. Please unify this to be the same in whole file.